examples/tracing/ddos_detector.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
```

Comment:
could you change this to `#!/usr/bin/python`? Sorry if this is what I suggested. We just formalized that all python script shebang should be `#!/usr/bin/python` instead of `#!/usr/bin/env python`.

Reply:
yes, done!